pulpanalytics/models.py

```diff
@@ -9,6 +9,11 @@ class System(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     first_seen = models.DateTimeField()
     postgresql_version = models.PositiveIntegerField()
+    users = models.PositiveIntegerField(default=0)
```

Comment:
I suspect these need to allow `null` so that systems that "dont have domains as even a feature" can be identified as `null`.